app/payload/control_freak/flight-software/src/data.h

```diff
@@ -0,0 +1,67 @@
+#ifndef FREAK_GORBFS_H
+#define FREAK_GORBFS_H
+#include <stdbool.h>
+#include <stdint.h>
+struct acc_data {
+    float ax; // m/s^2
+    float ay; // m/s^2
+    float az; // m/s^2
+};
+struct gyro_data {
+    float gx; // rad/s
+    float gy; // rad/s
+    float gz; // rad/s
+};
```

Comment:
NTypes provide these.

Reply:
Yes. See slack. TLDR: Standardization and post flight data handling infrastructure